src/new/wallet.js

```diff
@@ -256,8 +256,7 @@ class HathorWallet extends EventEmitter {
    * @memberof HathorWallet
    * @inner
    **/
-  async changeServer(newServer) {
-    await this.stop({cleanStorage: true, cleanAddresses: true});
+  changeServer(newServer) {
```

Comment:
There was no reason at all for stopping it? Won't we experience any drawback?

The code is very simple, but I don't have enough context right now to understand the real consequences of this change, that's why I'm asking. If we have a concrete motivation for doing this change, it could help in the contextualization as well.